garden/main.css

```diff
@@ -1,2270 +1,76 @@
-/*
-! tailwindcss v3.4.3 | MIT License | https://tailwindcss.com
-*/
+@tailwind base;
```

Comment:
The `@tailwind base`, `@tailwind components` and `@tailwind utilities` directives will fill in the necessary css style rules